README.md

```diff
@@ -259,3 +259,17 @@ This will run the `detect` phase of NAB on the data files specified in the above
 JSON file. Note that scoring and normalization are not supported with this
 option. Note also that you may see warning messages regarding the lack of labels
 for other files. You can ignore these warnings.
+
+##### Parameter Optimization on NAB
+
+You can run parameter optimization using your own framework or the framework provided by [htm.core](https://github.com/htm-community/htm.core). As of now, this is only enabled for the htm.core detector, but the same can be done for any detector with low effort (see #792 for details).
+**Requirements**
++ Docker Desktop
++ `pip install docker`
+
+**Usage**
+1. Set `use_optimization = True` in the htm.core detector settings.
+2. Build a docker image from the Dockerfile provided in this repo with `docker build -t optimize-htmcore-nab:latest . -f htmcore.Dockerfile`
+3. Then:
+    + Option A: Check `optimize_bayesopt.py` for an example on how to run with Bayesian Optimization. Note: The script requires `pip install bayesian-optimization`.
+    + Option B: Check `optimize_swarm.py` for an example on how to run the htm.core optimization framework. You can execute the script using the optimization framework with e.g. `python -m htm.optimization.ae -n 3 --memory_limit 4 -v --swarming 100 optimize_anomaly_swarm.py`. Note for MacOS users: You need to `export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES` before running the script. 
```

Comment:
about Note of OSX: is it something that's a problem in general? We could do it in the scripts in HTM repo already

Reply:
The problem comes from `docker.from_env()` where we fork a lot of processes at the same time. So this is just a problem that arises when using the optimization script.